docs/building-a-form/available-form-features-and-usage-guidelines.md

```diff
@@ -419,7 +419,17 @@ For the code implementation, see the [`review` folder](../../src/js/review).
 
 ### Required checkbox before form submission
 
-Use this feature to require a user to indicate they have read terms and conditions, a privacy policy, or any other text before submitting your form. This includes a checkbox and short-form text that can include relevant links to more verbose text on separate pages on your site.
+Use this feature to require a user to indicate they have read terms and conditions, a privacy policy, or any other text before submitting your form. It optionally includes a checkbox and short-form text that can include relevant links to more verbose text on separate pages on your site. To configure this feature, place a `preSubmitInfo` object in the `formConfig`:
```

Comment:
> It optionally includes a checkbox and short-form text

Which part of this is optional? By my understanding, the checkbox itself was already optional, but now the text that accompanies it is what's customizable, is that correct?

Reply:
Before this PR the checkbox was required and it had to be checked before the user could submit successfully. Also, the notice was hard-wired text that couldn't be changed by the user. 

With this PR the user can change the notice text or leave it out completely (by not specifying the `notice` property), and have a mandatory checkbox+label or leave it out completely (by setting `required: false` or leaving out the `required`/`field`/`label` properties entirely). The two features can be used independently, so you can just have a notice over the submit button for example that says "By submitting this form I agree to the privacy policy and represent that all the information here is true to the best of my knowledge."

I just added the `required` field this go-round and it brings up some unusual error cases because you can now say `required: true` but what if you don't specify the `label` or `field`? For now just pick some defaults to keep the ball rolling but this is more of a form developer error and we haven't addressed those issues although #176 is open to address such things.

I chose the settings in the example config to match the screen shot so we wouldn't need to shoot it again, but I'll need to verify that is actually true and there aren't any layout issues or whatever.

Reply:
I don't think we need the "optionally" here, then. Unfortunately, as it was originally written, it already seemed like the feature was optional. We can just go with:

> Use this feature to require a user to indicate they have read terms and conditions, a privacy policy, or any other text before submitting your form. It includes a checkbox and short-form text that can include relevant links to more verbose information on separate pages on your site. To configure this feature, place a `preSubmitInfo` object in the `formConfig`: